src/components/Combobox/components/ComboboxOption/ComboboxOption.jsx

```diff
@@ -124,10 +124,8 @@ const ComboboxOption = ({
       <div
         ref={ref}
         key={id || label}
-        role="row"
```

Comment:
not the correct row and makes issued with lint because of it. will consult with talya and fix in later pr